package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Reverse geocoding for air quality stations",
   "main": "eea-locations.js",
   "scripts": {
-    "start": "babel-node --presets es2015 eea-locations.js"
+    "battuta": "babel-node --presets es2015 eea-locations.js",
```

Comment:
@jflasher - the command I'm using in the Travis file is the one you are commenting on here so I think it's working! If not I can always try a global install command in the `before_install` portion of the file.